.github/workflows/standard_spa_pr.yml

```diff
@@ -95,12 +95,16 @@ on:
         required: false
       CYPRESS_RECORD_KEY:
         description: "The record key associated with the project in Cypress"
+        required: false
```

Comment:
We should be able to delete this yarn workflow now. We can update it to be safe and I will follow up in a different ticket

Reply:
thank you!